content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -123,7 +123,7 @@ different Kubernetes components.
 | `HPAScaleToZero` | `false` | Alpha | 1.16 | |
 | `HugePageStorageMediumSize` | `false` | Alpha | 1.18 | 1.18 |
 | `HugePageStorageMediumSize` | `true` | Beta | 1.19 | |
-| `IPv6DualStack` | `false` | Alpha | 1.15 | |
```

Comment:
The above line is supposed to be updated rather than dropped so people know the period in which the feature was Alpha.

```
| `IPv6DualStack` | `false` | Alpha | 1.15 | 1.15 |
```